### PR TITLE
feat: now managing sqs processor

### PR DESCRIPTION
**Pull request by simlu**

### CODE REVIEW

File: src/plugin/tasks/serverless/serverless-s3-api.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "target": "serverless/s3/api/${bucket}.yml",
+  "target": "serverless/api/s3/${bucket}.yml",
```
Comment (krackerman) [inline src/plugin/tasks/serverless/serverless-s3-api.json:2]: Should this file be renamed to account for this change? `serverless-api-s3.json`?

Comment (simlu) [reply src/plugin/tasks/serverless/serverless-s3-api.json:2]: It's not consistent... but it makes it easier to see the grouping

Comment (simlu) [reply src/plugin/tasks/serverless/serverless-s3-api.json:2]: But maybe we should jsut stick with that... let me see

**Pull request merged**